imagelab/ImgProvider.java

```diff
@@ -18,15 +18,14 @@
 public class ImgProvider extends JComponent {
     /** Serialization version. */
     private static final long serialVersionUID = 11L;
-
```

Comment:
Is this an intentional deletion of the line?  It seems like there might have been an appropriate intent to add a Javadoc comment rather than delete the line entirely,

Reply:
I added a JavaDoc to that line. It will be the same as one coming from ImageLabPackageCheckstyle.